solution/ui/e2e/cypress.json

```diff
@@ -2,5 +2,6 @@
   "baseUrl" : "http://localhost:8000",
   "videoUploadOnPasses": false,
   "defaultCommandTimeout": 10000,
-  "retries": 2
+  "retries": 2,
+  "chromeWebSecurity": false
```

Comment:
fixes cross-origin cypress bug that has been popping up on prod deployment recently.